packages/geoprocessing/scripts/dataPrep/bundleFeatures.ts

```diff
@@ -94,11 +95,13 @@ const bundleFeatures = async (
 
     // Fetch the bbox, id, and size (st_npoints) of all features to organizing
     // features into bundles
-    const records = await connection.many<{
-      id: number;
-      npoints: number;
-      bbox: BBox;
-    }>(sql`
+    const recordObject = z.object({
+      id: z.number(),
+      npoints: z.number(),
+      bbox: z.number().array().length(4).or(z.number().array().length(6)),
+    });
+
+    const records = await connection.many(sql.type(recordObject)`
```

Comment:
slonik is passed the `type` of the query result a different way now.  You pass in a zod object and it should actually now validates the results to match the type at runtime.